KarhooUISDKTests/TestCases/Screens/BookingScreen/QuoteList/KarhooQuoteListPresenterSpec.swift

```diff
@@ -100,18 +101,16 @@ class KarhooQuoteListPresenterSpec: XCTestCase {
     /**
      * Given: Prebook Quote search refreshes
      * When: Quote search finishes with results
-     * Then: Date formatter should been called and have correct time zone
+     * Then: Date should be set and have correct time zone
      * And: QuoteSorter should hide
      */
     func testSuccessFetchOfPrebook() {
         simulateSuccessfulQuoteFetch()
 
         let bookingDetails = mockBookingStatus.bookingDetailsToReturn
-        XCTAssertEqual(bookingDetails?.scheduledDate,
-                       mockDateFormatter.detailStyleDateSet)
+        XCTAssertNotNil(bookingDetails?.scheduledDate)
 
-        XCTAssertEqual(bookingDetails?.originLocationDetails?.timezone(),
-                       mockDateFormatter.timeZoneSet)
+        XCTAssertEqual(bookingDetails?.originLocationDetails?.timezone().identifier, "Europe/London")
```

Comment:
Let's move the `bookingDetailsDestinationSet`let from `simulateDestinationSetInBookingDetails `to the class level and rename it to `mockBookingDetails`. Then compare the `bookingDetails?.originLocationDetails?.timezone().identifier` with `mockBookingDetails.[...].timezone` instead of comparing it with the hardcoded value (that may change in `TestUtil.swift` at some point)